tools/internal/github/pr.go

```diff
@@ -15,7 +15,7 @@ import (
 // Client is a GitHub API client that performs PSL-specific
 // operations. The zero value is a client that interacts with the
 // official publicsuffix/list repository.
-type Client struct {
+type GHRepo struct {
```

Comment:
Tiny coding style suggestion: could we make this just `Repo`, or does that still cause confusion with the github library?

In general in Go, we try to name the types so that they are nice to read when combined with the package name, because every usage of the type requires the package. `github.GHRepo` repeats itself slightly, compared to `github.Repo`.

This is just an extremely minor suggestion, to be clear. GHRepo is also fine if you prefer that :)